doc/how_to/migrate/anywidget/index.md

```diff
@@ -0,0 +1,163 @@
+# Migrating from AnyWidget to Panel
```

Comment:
I hope you can find the time to systematically go through this `AnyWidget` migration guide and change the Panel `api` where it makes sense to minimize the number of steps it takes to migrate. That would build a bride between the Jupyter/ AnyWidget and Panel communities making collaboration easier.